R/calculate_vp.R

```diff
@@ -388,7 +392,7 @@ calculate_vp <- function(file, vpfile = "", pvolfile_out = "",
   )
 
   # read output into a vp object
-  output <- read_vpfiles(profile.tmp)
+  output <- read_vpts(profile.tmp)
```

Comment:
output should be a vp object, not vpts, even when the user has specified a .csv file as output.

So you should keep `read_vpfiles()`, and modify the output format from vpts in the case of a .csv file output. You can do that quite easily by subsetting the output format: when profile.tmp is a csv file, you will get a vpts object of lenght 1 from `read_vpfiles()`, which you can convert to vp using `output[1]`. bioRad has the [] operator overloaded, such that it can subset vpts objects.

Reply:
There is also the helper function `vpts_to_vp()` which allows us to keep `read_vpts()` as is and then convert back to `vp`

Reply:
Addressed in e97ef0768f19906f14781edbaf41dd1bbb83e2b9